libevm/VM.cpp

```diff
@@ -1591,6 +1622,8 @@ void VM::interpretCases()
             // bytes to handle "out of code" push data here.
             for (++m_PC; numBytes--; ++m_PC)
                 m_SPP[0] = (m_SPP[0] << 8) | m_code[m_PC];
+
+            trace();
```

Comment:
In this case `m_PC` is updated already, should this use a temporary variable for it too?

Reply:
I think it should

Reply:
Would be nice to move this `trace()` call in a common place... 
Can you try not to throw the exception immediately, just remember the error, then trance the instruction in a common place, and then when error throw it.

Reply:
Cannot one just call `trace` before the jumptable statement?

Reply:
Why not save PC as `startPC`before the jumptable and just call `trace` after it with the new  stack but the old PC?